.github/workflows/post-release.yml

```diff
@@ -36,10 +36,6 @@ jobs:
         uses: octokit/request-action@v2.x
         with:
           route: POST /repos/{owner}/{repo}/merges
-          owner: lichtblick-suite
```

Comment:
Are you sure that it will work? Can you attach some test or documentation confirming that? I checked on the official documentation, and they are still using thins parameters:

https://github.com/octokit/request-action